containers/EmpSponsors.ts

```diff
@@ -116,7 +136,43 @@ const useEmpSponsors = () => {
             };
           });
 
+          empData.liquidations.forEach((liquidation: LiquidationQuery) => {
+            const sponsor = utils.getAddress(liquidation.sponsor.id);
+            const liquidationCreatedEvent = liquidation.events.find(
+              (e) => e.__typename === "LiquidationCreatedEvent"
+            );
+            if (liquidationCreatedEvent) {
+              const liquidatedCR =
+                parseFloat(liquidation.tokensLiquidated) > 0
+                  ? parseFloat(liquidation.liquidatedCollateral) /
+                    parseFloat(liquidation.tokensLiquidated)
+                  : 0;
+              const maxDisputablePrice =
+                parseFloat(liquidation.tokensLiquidated) > 0 &&
+                collReqFromWei > 0
+                  ? parseFloat(liquidation.liquidatedCollateral) /
+                    (parseFloat(liquidation.tokensLiquidated) * collReqFromWei)
+                  : 0;
+              // There should always be a LiquidationCreatedEvent associated with each liquidation object, but if not then
+              // we will just ignore this strange edge case.
```

Comment:
nit: should this comment be at the top of this if statement?

Reply:
yeah definitely